benchmark/benchmark.mbt

```diff
@@ -0,0 +1,81 @@
+// Copyright 2024 International Digital Economy Academy
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+struct Task {
+  name : String
+  f : () -> Unit
+  count : Int
+}
+
+struct TaskResult {
+  task : Task
+  average : Double
+  max : Double
+  min : Double
```

Comment:
Are these fields expected to be private?

Reply:
No, maybe pub is better.

Reply:
I realized something, it seems that these fields can also be accessed when TaskResult is not set to pub.